### DISCUSSION TITLE
검색어 기능 만들어 보실 분?

### DISCUSSION THREAD

**Discussion by cocobabb** [codeCommonsLab/dev_doc_web#13] [Ideas]
https://github.com/user-attachments/assets/694c5a4c-3938-4793-8733-2a1b2ac06ec9

- 게시물 제목을 검색하면 검색어를 포함하는 게시글이 검색창 밑으로 표시되어야 합니다.
- 밑에 표시된 게시글을 클릭하면 해당 게시글 detail창으로 넘어갑니다.
- 검색어를 지우면 밑에 표시되던 게시글 목록이 사라져야 합니다.

**Comment by apricity2u** [comment]
저 기능 구현 후 pr 했습니다! 시간 되시는 분들은 코드 리뷰 부탁드려요:)

**Comment by kska01** [comment]
저도 구현해서 PR 했습니다!!! 
시간 되시는 분들은 코드 리뷰 부탁드립니다!! :)
